includes/class-simple-local-avatars.php

```diff
@@ -503,16 +504,28 @@ public function get_default_avatar_url( $size ) {
 	}
 
 	/**
-	 * Register admin settings.
+	 * Define the ratings avatar ratings.
+	 *
+	 * The ratings need to be defined after the languages have been loaded so
+	 * they can be translated. This method exists to define the ratings
+	 * after that has been done.
+	 *
+	 * @since x.x.x
 	 */
-	public function admin_init() {
+	public function define_avatar_ratings() {
 		$this->avatar_ratings = array(
 			'G'  => __( 'G &#8212; Suitable for all audiences', ),
```

Comment:
@peterwilsoncc Can we make these strings translatable?

Reply:
Not the keys, they're used by WP for determining the rating of an avatar. The values are the strings displayed to the user.

It's the equivalent of this code in core. https://github.com/WordPress/wordpress-develop/blob/1111d2b9e6d71bb5506e18940518cc96adbbdb97/src/wp-admin/options-discussion.php#L249-L258

Reply:
@peterwilsoncc Can you add a comment? I can see someone can make them translatable.

Reply:
* c75208f95a3fddcc84b5a5dfa295ddb261f76051 Note explaining why the key is note translated
* 9f8bd3ddbb21154b003fa005d70b09f6f2a85649 Added some translator comments duplicated from the same array in Core while I was dealing with i18n issues.